ha_lineairdb.hh

```diff
@@ -73,6 +74,7 @@ class LineairDB_share : public Handler_share {
   Class definition for the storage engine
 */
 class ha_lineairdb : public handler {
+  THD* thread;
```

Comment:
One handler is created per one connection.
`ha_innobase` has a member `THD *m_user_thd` as well.

Reply:
Okay. Thanks.
One more thing; I recommend renaming with a trailing underscore to clarify it is a class member. use `thread_`.

https://google.github.io/styleguide/cppguide.html#Variable_Names
> Data members of classes, both static and non-static, are named like ordinary nonmember variables, but with a trailing underscore.

Reply:
Understood.
I modified my vscode to color member variables so that I wouldn't need underscores, but if google recommends it, then I will😅
I read about coloring member variables in Clean Code.

Reply:
> I modified my vscode to color member variables so that I wouldn't need underscores, but if google recommends it, then I will😅

😆 

Coloring is a good practice. A certain number of people consider it a bad practice to indicate variables' roles by a notation, such as Hungarian notation or trailing underscores.

Maybe this is a problem of naming in a sense. The name `thread` looks like a local variable, not a class member.

Reply:
> Maybe this is a problem of naming in a sense. The name thread looks like a local variable, not a class member.
True. I will change it to `userThread`. 
For now, I will not use a trailing underscore.